src/esssans/metadata.py

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
+
+from .types import RawData, RunNumber, RunTitle, SampleRun
+
+
+def run_number(raw_data: RawData[SampleRun]) -> RunNumber:
+    """Get the run number from the raw sample data."""
+    return RunNumber(int(raw_data['run_number']))
+
+
+def run_title(raw_data: RawData[SampleRun]) -> RunTitle:
+    """Get the run title from the raw sample data."""
+    return RunTitle(raw_data['run_title'].value)
```

Comment:
I think those names a Mantid-specific, so maybe this should move into `sans2d`. NeXus has this information as well, but probably under slightly different names (I think `title` and `entry_identifier` in NXentry).

Reply:
Good point. I'll move it.